doc/reporter-mailx.txt

```diff
@@ -46,39 +46,42 @@ Lines can be continued on the next line using trailing backslash.
 
 Format:
 
-   "%summary:: subject format"
-   "section:: element1[,element2]..."
-   The literal text line to be added to email. Can be empty.
-   (Empty lines are NOT ignored!)
-
-   Subject format is a line of text, where %element% is replaced by
-   text element's content, and [[...%element%...]] block is used only if
-   %element% exists. [[...]] blocks can nest.
-
-   Sections can be:
-   - %summary: email subject format string.
-   - %attach: a list of elements to attach.
-   - text, double colon (::) and the list of comma-separated elements.
-
-   Elements can be:
-   - problem directory element names, which get formatted as
-     <element_name>: <contents>
-     or
-     <element_name>:
-     :<contents>
-     :<contents>
-     :<contents>
-   - problem directory element names prefixed by "%bare_",
-     which is formatted as-is, without "<element_name>:" and colons
-   - %oneline, %multiline, %text wildcards, which select all corresponding
-     elements for output or attachment
-   - %binary wildcard, valid only for %attach section, instructs to attach
-     binary elements
-   - problem directory element names prefixed by "-",
-     which excludes given element from all wildcards
-
-     Nonexistent elements are silently ignored.
-     If none of elements exists, the section will not be created.
+....
+%summary:: subject format
+section:: element1[,element2]...
+....
+
+The literal text line to be added to email. Can be empty.
+(Empty lines are NOT ignored!)
+
+Subject format is a line of text, where %element% is replaced by
+text element's content, and [[...%element%...]] block is used only if
+%element% exists. [[...]] blocks can nest.
+
+Sections can be:
+- %summary: email subject format string.
+- %attach: a list of elements to attach.
+- text, double colon (::) and the list of comma-separated elements.
+
+Elements can be:
+- problem directory element names, which get formatted as
+  <element_name>: <contents>
+  or
+  <element_name>:
+  :<contents>
+  :<contents>
+  :<contents>
+- problem directory element names prefixed by "%bare_",
+  which is formatted as-is, without "<element_name>:" and colons
+- %oneline, %multiline, %text wildcards, which select all corresponding
+  elements for output or attachment
+- %binary wildcard, valid only for %attach section, instructs to attach
+  binary elements
+- problem directory element names prefixed by "-",
+  which excludes given element from all wildcards
+
+Nonexistent elements are silently ignored.
+If none of elements exists, the section will not be created.
 
```

Comment:
The format part in the man page should look like format example in the following conf file https://github.com/abrt/libreport/blob/master/src/plugins/bugzilla_format.conf#L4 , moving the four dots here should fix the issue. Otherwise good job, I didn't know the four dots keyword.